src/App.vue

```diff
@@ -451,8 +460,8 @@ export default class App extends Mixins(BcolMixin, DateMixin, FilingTemplateMixi
         return
       }
       // if we get this far, the NR is good to go!
-      nrResponse.isConsumable = true
-      this.setNameRequestState(this.generateNameRequestState(nrResponse, filing.Id))
+      const nameRequestState = this.generateNameRequestState(nrResponse, filing.Id)
+      this.setNameRequestState(nameRequestState)
```

Comment:
I did this for temporary testing, and then it seemed cleaner to leave it like this :)